.travis.yml

```diff
@@ -10,5 +10,6 @@ before_script:
   - npm install -g grunt-cli
 
 script:
+  - commitlint-travis
```

Comment:
What's the advantage of also triggering this in CI?  Won't husky solve this?

I'm not necessarily against this, but I would like all the karma-runner projects to be similar if possible.

Reply:
@johnjbarton 
husky - works on developer side and check could be skip by "git commit  --no-verify" that mean server side should be verify second time on PRs build.

I included `commitlint-travis` it is similar that you did in karma 
```
matrix:
  fast_finish: true
  include:
    - name: "Lint code and commit message format"
      node_js: "10"
      env: VALIDATE_COMMIT_MSG=true LINT=true
```
but it didn't execute additional thread with the same execution tests

Reply:
My suggestion to use @commitlint/travis-cli and have less hacks in .travis.yml file

Reply:
@johnjbarton  any suggestions?

Reply:
Sorry got wrapped in trying to get the CI working on karma-runner.

I don't think `--no-verify` is too terrible for commit messages, worst case they don't trigger a release. But the check is also harmless if the commit is correct, so ok.